doc: Event-Based Sync for AWS Plugin (#13357)
cloudquery/cloudquery: website/pages/docs/plugins/sources/aws/configuration.mdx

          - <GetMetricStatistics>
    aws_cloudtrail_events:
      lookup_events:
        - <LookupEvents>
    aws_inspector2_findings:
      list_findings:
        - <ListFindings>
    aws_securityhub_findings:
      get_findings:
        - <GetFindings>
    aws_ecs_cluster_tasks:
      list_tasks:
        - <ListTasks>
  ```
    - [ListFindings](https://docs.aws.amazon.com/access-analyzer/latest/APIReference/API_ListFindings.html)
    - [ListMetrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_ListMetrics.html)
    - [GetMetricStatistics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html)
    - [LookupEvents](https://docs.aws.amazon.com/awscloudtrail/latest/APIReference/API_LookupEvents.html)
    - [ListFindings](https://docs.aws.amazon.com/inspector/v2/APIReference/API_ListFindings.html)
    - [GetFindings](https://docs.aws.amazon.com/securityhub/1.0/APIReference/API_GetFindings.html)
    - [ListTasks](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ListTasks.html)

> - **enterprise version only** `streaming_sync` (`array`) (default: empty)

Reviewer: ```suggestion
- **enterprise version only** `event_based_sync` (`array`) (default: empty)
```

  List of OU member accounts to skip. This is useful in conjunction with `organization_units` if there are accounts under the selected OUs that should be ignored.

> ### event_based_sync

Reviewer: ```suggestion
### event_based_sync

Event-based syncing is only supported by the enterprise-version AWS plugin. [Read more about event-based syncs]([/docs/plugins/sources/aws/event-based-sync](https://github.com/cloudquery/cloudquery/pull/13357/files#diff-d6b3e143b73c78cee7e26fed26091b5d29df661282616ba5ba58f010dadee2be)).
```

Reviewer: (I think we should just re-iterate that this is enterprise-only, in case users didn't see it in the section higher up)

- `account` ([Account](#account)) 

  Configuration for the credentials that will be used to grab records from the specified Kinesis Stream. If this is not specified the default credentials will be used

- `kinesis_stream_arn` (string) (**required**)

  ARN for the Kinesis stream that will hold all of the CloudTrail records

- `start_time` (timestamp) 

  Defines the place in the stream where record processing should begin. By default, the time at which the sync began will be used.

- `event_based_sync_only` (`bool`) (default: `false`)

>   By default CQ will do a full sync on the specified tables before starting to consume the events in the stream. This parameter enables users to skip the pull based sync and go straight to the streaming sync.

Reviewer: ```suggestion
  By default CQ will do a full sync on the specified tables before starting to consume the events in the stream. This parameter enables users to skip the pull-based sync and go straight to the streaming sync.
```

  List of OU member accounts to skip. This is useful in conjunction with `organization_units` if there are accounts under the selected OUs that should be ignored.

{/* <!-- vale off --> */}
### event_based_sync
{/* <!-- vale on --> */}

<Callout type="info">
Event-based syncing is only supported by the enterprise-version AWS plugin. Read more about [event-based syncs](/docs/plugins/sources/aws/event-based-sync)
</Callout>

- `account` ([Account](#account)) 

  Configuration for the credentials that will be used to grab records from the specified Kinesis Stream. If this is not specified the default credentials will be used

- `kinesis_stream_arn` (string) (**required**)

  ARN for the Kinesis stream that will hold all of the CloudTrail records

- `start_time` (timestamp) 

  Defines the place in the stream where record processing should begin. By default, the time at which the sync began will be used.

> - `event_based_sync_only` (`bool`) (default: `false`)

Reviewer 2: ```suggestion
- `full_sync` (`bool`) (default: `true`)
```

>   By default CQ will do a full sync on the specified tables before starting to consume the events in the stream. This parameter enables users to skip the pull-based sync and go straight to the streaming sync.

Reviewer 2: ```suggestion
  By default CQ will do a full sync on the specified tables before starting to consume the events in the stream. This parameter enables users to skip the full sync and go straight to the event based sync.
```
